### ISSUE TITLE
Remind TiCs to add post-event notes

### ISSUE THREAD

**Issue by NoRePercussions** [ABTech/tracker#550]
Tech currently has a policy that TiCs should add billing-related notes to the comments of each event. However, this is often forgotten after an event.

Tracker should send an email some time after an event completes that reminds TiCs to do so, potentially conditional on whether they've commented since the event ended.

Wellington also noted that we could send Slack notifications. It looks like Tracker is a bot, so we could roughly associate Tracker users and Slack users by email/phone. I'm not sure if that should be broken out into another issue.

**Comment by wjiang42** [comment]
The idea was @maxjkwang's  lol

I think a good reminder time would be 24 hours after strike begins, since all relevant ends should have a strike time, and few events have strikes longer than a day.

**Comment by DaAwesomeP** [comment]
This can probably build off of the already implemented feature that marks events as completed to schedule the job.

There should be an event checkbox (checked by default) to enable this. We should pick some nice time to notify on the following day (9am?) in case strike ends at midnight.

This one definitely requires #544.

**Comment by DaAwesomeP** [comment]
For Slack DMing, yes maybe a separate issue to make a method to send a notification to one or more people. I believe we can lookup users by email. Rob has played around with this and we can ask him in a separate thread.

**Comment by maxjkwang** [comment]
Yeah my initial thoughts were to notify the following noon, but I guess 9am is also fine, not a big problem.

If it's easy to implement then that's nice.
